src/contractTest/java/uk/gov/hmcts/reform/em/orchestrator/provider/ProviderTestUtil.java

```diff
@@ -54,6 +56,26 @@ public static CcdCallbackResponseDto createNewBundleResponse() {
         return response;
     }
 
+    public static CcdCallbackResponseDto createCloneBundleResponse() {
+        ObjectNode originalBundle = buildCcdBundle();
+        originalBundle.put("eligibleForCloning", "no");
+
+        ObjectNode clonedBundle = buildCcdBundle();
+        clonedBundle.put("id", UUID.randomUUID().toString()); // It gets a new ID
+        clonedBundle.put("title", "CLONED_" + originalBundle.get("title").asText());
+        clonedBundle.put("fileName", "CLONED_" + originalBundle.get("fileName").asText());
+        clonedBundle.put("eligibleForCloning", "no");
```

Comment:
mismatch here due to "no" -> body: $.data.caseBundles[0].value.eligibleForCloning Expected 'no' to match 'Yes|No'